spec/jobs/notify_slack_spec.rb

```diff
@@ -17,6 +19,29 @@
       response = Jobs::NotifySlack.new.execute({post_id: post[:id]})
       expect(response[0]).to eq("success")
     end
+
+    it 'should send notification only for particular category topic' do
```

Comment:
Hmm I don't think the description is clear in explaining what we're testing for here. Do you mean that we're testing that the job should only send a notification for topics in a category which has a filter set?